docs/uds-packages/guidelines/metadata-guidelines.md

```diff
@@ -1,7 +1,40 @@
 
-Instruction on metadata will be included in the uds packages [Template README](https://github.com/uds-packages/template/blob/main/README.md)
\ No newline at end of file
+> [!NOTE]
+> Annotations are added to your `zarf.yaml` under the `metadata.annotations` key:
+> ```yaml
+> kind: ZarfPackageConfig
+> metadata:
+>   annotations:
+>     dev.uds.title: "Example Package"
+>     dev.uds.categories: "Development, Testing"
+>     # Additional annotations as needed
+> ```
+
+## Production Ready Annotations
+
+The following annotations are production ready and you are encouraged to use them widely in your packages:
+
+| Annotation | Description | Example | Products |
+| --- | --- | --- | --- |
+| `dev.uds.title` | The human readable title for the package | `"GitLab Runner"` | [UDS Registry](https://github.com/defenseunicorns/uds-registry/blob/main/docs/development/development.md#zarf-package-metadata) |
+| `dev.uds.categories` | A list of dynamic categories the package belongs to | `"Software Dev, IT Management, Kubernetes (K8s), Productivity"` | [UDS Registry](https://github.com/defenseunicorns/uds-registry/blob/main/docs/development/development.md#zarf-package-metadata) |
+| `dev.uds.keywords` | A list of search keywords associated with the package | `"CI/CD Pipelines, Automation, GitLab Integration"` | [UDS Registry](https://github.com/defenseunicorns/uds-registry/blob/main/docs/development/development.md#zarf-package-metadata) |
+| `dev.uds.tagline` | A tagline for the package | `"Open Source CI/CD Job Execution Tool"` | [UDS Registry](https://github.com/defenseunicorns/uds-registry/blob/main/docs/development/development.md#zarf-package-metadata) |
+| `dev.uds.icon` | A base64 encoded icon for the package represented as a [data URI](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URIs) | `"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAADUlEQVR42mNkYAAAAAYAAjCB0C8AAAAASUVORK5CYII="` | [UDS Registry](https://github.com/defenseunicorns/uds-registry/blob/main/docs/development/development.md#zarf-package-metadata) |
+
+> [!IMPORTANT]
+> Consider how these annotations impact user experience when your package appears in the UDS Registry. Keep information concise and descriptive, and optimize icons for both size and quality.
+
+## Provisional Annotations
+
+Below are annotations that are experimental and not ready for broader use (yet). You may still implement these however and feedback is appreciated.
+
+| Annotation | Description | Example | Products |
+| --- | --- | --- | --- |
+| `dev.uds.system` | Whether the package is a system package or an application package | `"true"` | [UDS Android](https://github.com/defenseunicorns/uds-android) |
+| `dev.uds.time.deploy.estimate` | An estimate of the time it will take to deploy the package represented as a [Go duration](https://pkg.go.dev/time#ParseDuration) | `"15m"` | [UDS Android](https://github.com/defenseunicorns/uds-android) |
```

Comment:
Should the link here be to a public docs page somewhere instead of the private repo?

Reply:
swapped